### PR TITLE
Update Mac .app support.

### PR DESCRIPTION
**Pull request by frank-trampe**
Standardize the paths for the Macintosh .app build scripts and document the process.

### CODE REVIEW

**Comment by davelab6** [comment]
@monkeyiq please can you review this?

File: osx/README [hunk 1]
```diff
@@ -3,35 +3,25 @@ become a little post processing step and the make-package.sh simply
 uses the create-osx-app-bundle.sh script to create an app.zip file
 from the macports built sources.
 
-As at April 2014 the build is performed using the following script
-sequence. The make-package script will create
-~/Desktop/FontForge.app.zip which you can then archive and upload that
++As of January 2015, the build is performed using the following script
```
Comment (davelab6) [inline osx/README]: Why do these lines start with `+`?

Comment (frank-trampe) [inline osx/README]: Bad copying and pasting. I'll fix it now.

**Comment by frank-trampe** [comment]
See #2139.

**Pull request closed without merging**